src/neptune/constants.py

```diff
@@ -39,3 +40,9 @@
 
 MAX_32_BIT_INT = 2147483647
 MIN_32_BIT_INT = -2147483648
+
+NAN_FLOAT = float("nan")
+INFINITY_FLOAT = float("inf")
+NEG_INFINITY_FLOAT = float("-inf")
+
+UNSUPPORTED_FLOATS = [NAN_FLOAT, INFINITY_FLOAT, NEG_INFINITY_FLOAT]
```

Comment:
Since this will (and never should) be modified, could we change it to a tuple?

Reply:
Changed it to function, to never miss a case 😇